doc/gitian-building.md

```diff
@@ -172,50 +170,94 @@ and proceed, just press `Enter`. To select a different button, press `Tab`.
 
 ![](gitian-building/debian_install_16_choose_a_mirror.png)
 
-- Enter proxy information (unless you are on an intranet, you can leave this empty)
+- Enter proxy information (unless you are on an intranet, leave this empty)
 
 ![](gitian-building/debian_install_18_proxy_settings.png)
 
 - Wait a bit while 'Select and install software' runs
 - Participate in popularity contest -> *No*
-- Choose software to install. We need just the base system. 
+- Choose software to install. We need just the base system.
+- Make sure only 'SSH server' and 'Standard System Utilities' are checked
+- Uncheck 'Debian Desktop Environment' and 'Print Server'
 
 ![](gitian-building/debian_install_19_software_selection.png)
 
-- Make sure only 'SSH server' and 'Standard System Utilities' are checked
-- Uncheck 'Debian Desktop Environment' and 'Print Server'
+- Install the GRUB boot loader to the master boot record? -> Yes
 
 ![](gitian-building/debian_install_20_install_grub.png)
 
-- Install the GRUB boot loader to the master boot record? -> Yes 
+- Device for boot loader installation -> ata-VBOX_HARDDISK
 
-![](gitian-building/debian_install_21_finish_installation.png)
+![](gitian-building/debian_install_21_install_grub_bootloader.png)
 
 - Installation Complete -> *Continue*
 - After installation, the VM will reboot and you will have a working Debian VM. Congratulations!
 
+![](gitian-building/debian_install_22_finish_installation.png)
+
+
+After Installation
+-------------------
+The next step in the guide involves logging in as root via SSH.
+SSH login for root users is disabled by default, so we'll enable that now.
+
+Login to the VM using username `root` and the root password you choose earlier.
+You'll be presented with a screen similar to this.
+
+![](gitian-building/debian_root_login.png)
+
+Type,
+
+```
+nano /etc/ssh/sshd_config
+```
+and press enter. You should see:
+
+![](gitian-building/debian_root_nano_sshd_config.png)
+
+Find the line (using the down key) that reads
+```
+PermitRootLogin without-password
+```
+and change it to
+```
+PermitRootLogin yes
+```
```

Comment:
Any reason we want to go through the process of editing with `nano` manually instead of just saying something like "Run `sed -i 's/^PermitRootLogin.*/PermitRootLogin yes/' /etc/ssh/sshd_config`"?